Adjust this test to musl and our output of 'ldd /bin/ls'.
This test will be run on x86_64.
--- a/src/dist/pkg.rs
+++ b/src/dist/pkg.rs
@@ -378,24 +378,28 @@ mod toolchain_imp {
     }
 
     #[test]
-    fn test_ldd_parse_v2_30() {
-        let archlinux_ls_output = "\tlinux-vdso.so.1 (0x00007ffddc1f6000)
-\tlibcap.so.2 => /usr/lib/libcap.so.2 (0x00007f4980989000)
-\tlibc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f69ca6a1000)
-\tlibc.so.6 => /usr/lib/libc.so.6 (0x00007f49807c2000)
-\t/lib64/ld-linux-x86-64.so.2 => /usr/lib64/ld-linux-x86-64.so.2 (0x00007f49809e9000)
+    fn test_musl_ldd_parse_v1_2_5() {
+        let alpinelinux_ls_output = "\t/lib/ld-musl-x86_64.so.1 (0x7f56f69bc000)
+\tlibcrypto.so.3 => /lib/libcrypto.so.3 (0x7f56f6474000)
+\tlibacl.so.1 => /lib/libacl.so.1 (0x7f56f646b000)
+\tlibattr.so.1 => /lib/libattr.so.1 (0x7f56f6464000)
+\tlibutmps.so.0.1 => /lib/libutmps.so.0.1 (0x7f56f645f000)
+\tlibc.musl-x86_64.so.1 => /lib/ld-musl-x86_64.so.1 (0x7f56f69bc000)
+\tlibskarnet.so.2.14 => /lib/libskarnet.so.2.14 (0x7f56f6427000)
 ";
         assert_eq!(
-            parse_ldd_output(archlinux_ls_output)
+            parse_ldd_output(alpinelinux_ls_output)
                 .iter()
                 .map(|p| p.to_str().unwrap())
                 .collect::<Vec<_>>(),
             &[
-                "/usr/lib/libcap.so.2",
-                "/lib/x86_64-linux-gnu/libc.so.6",
-                "/usr/lib/libc.so.6",
-                "/lib64/ld-linux-x86-64.so.2",
-                "/usr/lib64/ld-linux-x86-64.so.2",
+                "/lib/ld-musl-x86_64.so.1",
+                "/lib/libcrypto.so.3",
+                "/lib/libacl.so.1",
+                "/lib/libattr.so.1",
+                "/lib/libutmps.so.0.1",
+                "/lib/ld-musl-x86_64.so.1",
+                "/lib/libskarnet.so.2.14",
             ]
         )
     }
